app/models/order.rb

```diff
@@ -13,6 +13,8 @@ class Order < ActiveRecord::Base
     ]
   end
 
+  validates :booking_type_id, presence: true
```

Comment:
@bharat619 some orders (such as shipments) don't have a specific booking type so I'm not sure if we can require this field

cc @patrixr

Reply:
sure @steveyken ,
could you take a look now. Changes made as per our discussion